common/js/src/popup-window/in-popup-main-script.js

```diff
@@ -61,25 +64,26 @@ GSC.InPopupMainScript.showWindow = function() {
   chrome.app.window.current().show();
 };
 
+const lastUsedPopupId = GSC.InPopupMainScript.getData()['popup_id'];
```

Comment:
nit: A better name would be "currentPopupId" or "ourPopupId".

It's not necessarily a "lastUsedPopupId", since there might be multiple popups opened.